doc/modules/clustering.rst

```diff
@@ -338,15 +338,17 @@ to be the exemplar of sample :math:`i` is given by:
 
 .. math::
 
-    r(i, k) \leftarrow s(i, k) - max [ a(i, \acute{k}) + s(i, \acute{k}) \forall \acute{k} \neq k ]
+    r(i, k) \leftarrow s(i, k) - \max_{\overline k \ne k} \left( a(i, \overline
```

Comment:
I've never thought of the variable used in this scenario as different from any other variable, and I've used and seen others use overlines and tons of other things (but never acute accents, until now...) Anyway, I changed from the acute accent because I found it hard to distinguish with the dot on the i. I can change it if desired.